Horatio/Operations/AlertOperation.swift

```diff
@@ -38,8 +38,7 @@ class AlertOperation: Operation {
     // MARK: Initialization
 
     init(presentationContext: UIViewController? = nil) {
-        self.presentationContext = presentationContext ?? UIApplication.shared.keyWindow?.rootViewController
-
+        self.presentationContext = presentationContext
```

Comment:
The issue is, at the time that the actual alert is presented, there could be another VC already presented on the root VC.